src/js/components/Meter/__tests__/__snapshots__/Meter-test.tsx.snap

```diff
@@ -24,6 +24,7 @@ exports[`Meter background 1`] = `
   class="c0"
 >
   <svg
+    aria-label="bar meter with undefined% filled"
```

Comment:
What's happening with all the `undefined` substitution values?

Reply:
its when there is no `value` fixing now

Reply:
in one test we have <Meter /> without any `value` passed in

Reply:
There shouldnt be anymore undefined